Sources/Sentry/Public/SentryUserFeedback.h

```diff
@@ -0,0 +1,44 @@
+#import "SentryDefines.h"
+#import "SentrySerializable.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@class SentryId;
+
+/**
+ * Adds additional information about what happened to an event.
+ */
+NS_SWIFT_NAME(UserFeedack)
```

Comment:
`m`: Because we were talking about this at the standup, should we call this now also `SentryUserFeedback` in Swift? @bruno-garcia, @marandaneto and @jennmueng

Reply:
Given that UserFeedback is a more specific name, I have no strong feelings here.

I rather default to `SentryName` when the name is super generic like `Client` or `Event`.